public/styles/styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  background-image: url("./blue-background-with-text-overlay-1314536.jpg");
+  background-repeat: no-repeat;
+  background-size: auto;
+}
+button {
```

Comment:
Be consistent with your spacing, it can read as a lack of `attention to detail` which is never a good impression for a dev